### ISSUE TITLE
component suggestions

### ISSUE THREAD

**Issue by dobkeratops** [bbernhard/imagemonkey-core#210]
Hi, so i found searching  ```person.has='head'``` works fine now.. this is a great addition as it's more relaxed to draw boxes around a part rather than tracing the whole outlines.. its easy to dip in casually to do this

following on from this , would it be an idea to enable some more components?

suggestions:-
```hand/person``` ```person.has='hand'```
```foot/person``` ```person.has='foot'```

- with these two in addition to the head, you can get a box around most of the extremities 

```head/man```, ```head/woman``` .. these aliases could enable assigning gender via the head

```face/person``` .. subset of head.. and distinguishes the front or side of the head from the back

paw/dog,  paw/cat    ... (very common again)

tail 

car parts:-
```wheel```, ```headlight```,```tail light``` (useful to distinguish car orientation)

```cabin``` - some vehicles - trucks, tractors, boats..

miscellaneous - ```handle``` .. for knife, fork, and various tools (imagine if you could anotate ```handle/container``` or ```handle/tool```

**Comment by bbernhard** [comment]
Many thanks for the suggestions! :)

The following labels will be added with the next (probably mid/end of next week) batch: 

tail/cat
tail/dog
paw/dog
paw/cat
hand/person
foot/person
face/person
wheel/car
headlight/car

The other suggestions will follow :)

Does it make sense to also add a descriptive text for some of the labels? I guess labels like `face` <-> `head` could be easily mixed up when annotating stuff. (especially by the occasional annotator).

**Comment by dobkeratops** [comment]
> Does it make sense to also add a descriptive text for some of the labels? I guess labels like face <-> head could be easily mixed up when annotating stuff.

agree.. that might not be so obvious to the casual user. not sure what the best solution is, or which of those two is more obvious.. although I was going to make another suggestion of a 'label browser' showing confirmed good examples

**Comment by bbernhard** [comment]
the labels are now productive :) I was really surprised to see that those labels were used 1621 times - that's totally awesome! That's a lot more than I expected. WOW!

**Comment by dobkeratops** [comment]
yup. components are more relaxing to do..much easier to do casually
